app/controllers/admin/users/view/events/list.js

```diff
@@ -32,11 +32,11 @@ export default class extends Controller.extend(EmberTableControllerMixin) {
       {
         name            : 'Date',
         valuePath       : 'startsAt',
-        extraValuePaths : ['endsAt'],
+        extraValuePaths : ['endsAt', 'timezone'],
```

Comment:
Undo all changes irrelevant to this issue.

Reply:
This change shouldn't be here